api/clients/accountant.go

```diff
@@ -54,14 +54,14 @@ func NewAccountant(reservation *core.ActiveReservation, onDemand *core.OnDemandP
 		binRecords[i] = BinRecord{Index: uint32(i), Usage: 0}
 	}
 	a := accountant{
+		accountID:         accountID,
 		reservation:       reservation,
 		onDemand:          onDemand,
 		reservationWindow: reservationWindow,
 		pricePerSymbol:    pricePerSymbol,
 		minNumSymbols:     minNumSymbols,
 		binRecords:        binRecords,
 		cumulativePayment: big.NewInt(0),
-		paymentSigner:     paymentSigner,
 		numBins:           max(numBins, minNumBins),
 	}
 	// TODO: add a routine to refresh the on-chain state occasionally?
```

Comment:
The idea is that the accountant will track a fix number of bins (numBins), and overwrite them in a circular manner. The `relativeBinRecord = binRecord % numBins` so that we can identify the correct position on this array for the current bin; similar to overflowBinRecord, which is at `binRecord+2`